ci/pipeline.yaml

```diff
@@ -0,0 +1,59 @@
+resources:
+  - name: re-request-an-aws-account-git
+    type: git
+    source:
+      uri: https://github.com/alphagov/re-request-an-aws-account.git
+      branch: master
+
+  - name: deploy-to-paas-aws-account-management-space
+    type: cf
+    source:
+      api: https://api.cloud.service.gov.uk
+      username: ((cf_user))
+      password: ((cf_password))
+      organization: gds-tech-ops
+      space: re-aws-account-management
+jobs:
+  - name: self-update
+    serial: true
+    plan:
+    - get: re-request-an-aws-account-git
+      trigger: true
+    - set_pipeline: request-an-aws-account
```

Comment:
Y'know you can use the magic `self` value here which I think is a bit safer in case people try spinning up dev versions of this pipeline.